.gitignore copy

```diff
@@ -0,0 +1,105 @@
+# Logs
```

Comment:
what is the file purpose?

Reply:
i copied the work-standard into the existing folder and i already had a gitignore :(
fixed